store/authSlice.js

```diff
@@ -2,9 +2,21 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const authSlice = createSlice({
   name: 'auth',
-  initialState: { isAuth: false },
-  reducers: {},
+  initialState: {
+    isAuth: false,
+    user: { id: null, url: null, nickName: null },
+  },
+  reducers: {
+    checkAuth(state, action) {
+      state.isAuth = action.payload;
+    },
+    setUser(state, action) {
+      state.user.id = action.payload.user.id;
+      state.user.url = action.payload.user.url;
+      state.user.nickName = action.payload.user.nickName;
```

Comment:
구조분해할당을 활용해봐도 좋을 것 같습니다.

```suggestion
      const { id, url, nickName } = action.payload.user;
      state.user = { id, url, nickName }
```

url은 imageUrl로 명확히 표기하면 좋을 것 같습니다. nickname은 한단어이므로, 모두 소문자로 적어주는게 좋을 것 같습니다.

Reply:
이 부분도 구조분해할당을 이용하면 훨씬 가독성 있고 좋네요 명확한 표기명과 변수명에 대해서 더 깊이 생각하겠습니다